### PR TITLE
Release/4.4

### PR DESCRIPTION
**Pull request by kyranjamie**
> [Download the latest build](https://github.com/blockstack/stacks-wallet/actions/runs/1030368473)

### CODE REVIEW

File: app/components/home/transaction-list/transaction-list-empty.tsx [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 import { templateTxBoxProps } from './transaction-list-item-pseudo';
 
 export const TransactionListEmpty = () => {
   const { isOnline } = useNavigatorOnline();
   return (
     <Flex {...templateTxBoxProps}>
-      <Text textStyle="body.small" display="block" textAlign="center" mb="tight">
+      <Box maxWidth="180px" mb="loose">
+        <EmptyTxList />
+      </Box>
```
Comment (aulneau) [inline app/components/home/transaction-list/transaction-list-empty.tsx:14]: could be this
```suggestion
      <Box as={EmptyTxList} maxWidth="180px" mb="loose" />
```

File: app/constants/index.ts [hunk 2]
```diff
@@ -62,13 +62,15 @@ export const REVOKE_DELEGATION_TX_SIZE_BYTES = 165;
 
 export const STACKING_CONTRACT_CALL_FEE = 260;
 
-export const POOLED_STACKING_TX_SIZE_BYTES = 199;
+export const POOLED_STACKING_TX_SIZE_BYTES = 216;
```
Comment (aulneau) [inline app/constants/index.ts:65]: this never changes?

**Review by aulneau — APPROVED** [review]

File: app/constants/index.ts (hunk 2 above)
Comment (kyranjamie) [reply app/constants/index.ts:65]: So, I think I just got it wrong the first time. But could be proven wrong, if so then this implementation needs to change.

**Review by fbwoolf — APPROVED** [review]

File: app/components/home/transaction-list/transaction-list-empty.tsx (hunk 1 above)
Comment (kyranjamie) [reply app/components/home/transaction-list/transaction-list-empty.tsx:14]: noted but don't want to wait for ci to rebuild

**Pull request merged**